multiregion/README.md

```diff
@@ -342,20 +341,18 @@ Verify the new topic replica placement is restored.
 Sample output:
 
 ```
-==> Describe topic single-region
-
-Topic: single-region    PartitionCount: 1       ReplicationFactor: 2    Configs: min.insync.replicas=1,confluent.placement.constraints={"version":1,"replicas":[{"count":2,"constraints":{"rack":"west"}}],"observers":[]}
-        Topic: single-region    Partition: 0    Leader: 2       Replicas: 2,1   Isr: 1,2        Offline:        LiveObservers:
+Topic: single-region	PartitionCount: 1	ReplicationFactor: 2	Configs: min.insync.replicas=1,confluent.placement.constraints={"version":1,"replicas":[{"count":2,"constraints":{"rack":"west"}}],"observers":[]}
+	Topic: single-region	Partition: 0	Leader: 2	Replicas: 2,1	Isr: 1,2	Offline: 
 
 ==> Describe topic multi-region-sync
 
-Topic: multi-region-sync        PartitionCount: 1       ReplicationFactor: 4    Configs: min.insync.replicas=1,confluent.placement.constraints={"version":1,"replicas":[{"count":2,"constraints":{"rack":"west"}},{"count":2,"constraints":{"rack":"east"}}],"observers":[]}
-        Topic: multi-region-sync        Partition: 0    Leader: 1       Replicas: 1,2,3,4       Isr: 3,4,2,1    Offline:        LiveObservers:
+Topic: multi-region-sync	PartitionCount: 1	ReplicationFactor: 4	Configs: min.insync.replicas=1,confluent.placement.constraints={"version":1,"replicas":[{"count":2,"constraints":{"rack":"west"}},{"count":2,"constraints":{"rack":"east"}}],"observers":[]}
+	Topic: multi-region-sync	Partition: 0	Leader: 1	Replicas: 1,2,3,4	Isr: 3,4,2,1	Offline: 
 
 ==> Describe topic multi-region-async
 
-Topic: multi-region-async       PartitionCount: 1       ReplicationFactor: 4    Configs: min.insync.replicas=1,confluent.placement.constraints={"version":1,"replicas":[{"count":2,"constraints":{"rack":"west"}}],"observers":[{"count":2,"constraints":{"rack":"east"}}]}
-        Topic: multi-region-async       Partition: 0    Leader: 1       Replicas: 1,2,4,3       Isr: 2,1        Offline:        LiveObservers: 4,3
+Topic: multi-region-async	PartitionCount: 1	ReplicationFactor: 4	Configs: min.insync.replicas=1,confluent.placement.constraints={"version":1,"replicas":[{"count":2,"constraints":{"rack":"west"}}],"observers":[{"count":2,"constraints":{"rack":"east"}}]}
+	Topic: multi-region-async	Partition: 0	Leader: 2	Replicas: 2,1,3,4	Isr: 2,1	Offline: 	Observers: 3,4
```

Comment:
@ybyzek I think we should remove the last sentence in the note on line 363 below. What do you think?

Reply:
👍 removed